docs/community/tutorials-and-resources.md

```diff
@@ -101,12 +101,6 @@ Want your Django REST Framework talk/tutorial/article to be added to our website
 [django-rest-framework-course]: https://teamtreehouse.com/library/django-rest-framework
 [pycon-uk-2016]: https://www.youtube.com/watch?v=FjmiGh7OqVg
 [django-under-hood-2014]: https://www.youtube.com/watch?v=3cSsbe-tA0E
-[integrating-pandas-drf-and-bokeh]: https://machinalis.com/blog/pandas-django-rest-framework-bokeh/
-[controlling-uncertainty-on-web-apps-and-apis]: https://machinalis.com/blog/controlling-uncertainty-on-web-applications-and-apis/
-[full-text-search-in-drf]: https://machinalis.com/blog/full-text-search-on-django-rest-framework/
```

Comment:
[full-text-search-in-drf]: https://machinalis32.rssing.com/chan-67888834/latest.php#item31